src/read.c

```diff
@@ -367,6 +371,66 @@ static uint32_t avifCodecConfigurationBoxGetDepth(const avifCodecConfigurationBo
     return 8;
 }
 
+#if defined(AVIF_ENABLE_EXPERIMENTAL_SAMPLE_TRANSFORM)
+uint8_t avifCodecConfigurationBoxGetSubsamplingType(const avifCodecConfigurationBox * av1C, uint8_t channelIndex)
+{
+    // See ISO/IEC 23008-12:2024/CDAM 2:2025 section 6.5.6.3.
+    if (channelIndex == 0) {
+        return 0; // 4:4:4
```

Comment:
could you use an enum? just a private one in this file would make things clearer I think but up to you